eng/common/scripts/stress-testing/stress-test-deployment-lib.ps1

```diff
@@ -67,11 +67,18 @@ function DeployStressTests(
     [string]$repository = '',
     [switch]$pushImages,
     [string]$clusterGroup = '',
-    [string]$deployId = 'local',
+    [string]$deployId = '',
     [switch]$login,
     [string]$subscription = '',
     [switch]$CI,
-    [string]$Namespace
+    [string]$Namespace,
+    [ValidateScript({
+        if (!(Test-Path $_)) {
+            throw "LocalAddonsPath $LocalAddonsPath does not exist"
+        }
+        return $true
+    })]
+    [System.IO.FileInfo]$LocalAddonsPath
```

Comment:
So we're making localaddons path mandatory? I feel like it might be better if it's only mandatory if we can't auto-discover and throw an error if localAddonsPath is not provided && can't auto-discover it

Reply:
Correct. If you don't pass it in, it does not validate.